Neos.Flow/Scripts/flow.php

```diff
@@ -39,7 +39,17 @@
     array_shift($argv);
     require(__DIR__ . '/migrate.php');
 } else {
-    $composerAutoloader = require(__DIR__ . '/../../../Libraries/autoload.php');
+    $composerAutoloader = dirname(__DIR__, 3) . '/Libraries/autoload.php';
+    if (!file_exists($composerAutoloader)) {
+        echo 'Composers "autoload.php" file was not found. The file is expected to be located in the path:' . PHP_EOL . PHP_EOL;
+        echo $composerAutoloader . PHP_EOL . PHP_EOL;
+        echo 'This could be due to a missing "config" => "vendor-dir" section of your root "composer.json" file.' . PHP_EOL . PHP_EOL;
+        echo 'The section key and value should look like the following:' . PHP_EOL;
+        echo '"vendor-dir": "Packages/Libraries"' . PHP_EOL;
+        echo 'Update your "composer.json" file accordingly and run the "composer update" command.' . PHP_EOL;
+        exit(1);
+    }
+    require($composerAutoloader);
```

Comment:
```suggestion
    $autoloaderPath = dirname(__DIR__, 3) . '/Libraries/autoload.php';
    if (!file_exists($autoloaderPath)) {
        echo 'Composers "autoload.php" file was not found. The file is expected to be located in the path:' . PHP_EOL . PHP_EOL;
        echo $autoloaderPath . PHP_EOL . PHP_EOL;
        echo 'This could be due to a missing "config" => "vendor-dir" section of your root "composer.json" file.' . PHP_EOL . PHP_EOL;
        echo 'The section key and value should look like the following:' . PHP_EOL;
        echo '"vendor-dir": "Packages/Libraries"' . PHP_EOL;
        echo 'Update your "composer.json" file accordingly and run the "composer update" command.' . PHP_EOL;
        exit(1);
    }
    $composerAutoloader = require($autoloaderPath);
```

Reply:
When I read "instanciated" I was like "WTF? There wasn't `new` in the code before!". But yes, we definitely need to _assign_ it! 🤣

Reply:
oh man, sorry for the confusion 😆